### PR TITLE
Fix logits bias in API [completions.py]

### PR DESCRIPTION
**Pull request by Ph0rk0z**
## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/oobabooga/text-generation-webui/wiki/Contributing-guidelines).

When loading a model from the UI, generate_params['model'] is nonsense or empty and this cases tiktoken to fail like in this issue. After fixing it, I find that the bias is working well again. https://github.com/oobabooga/text-generation-webui/issues/5232

I have no clue about the other case where it's used so I can't test it, but if there is a model generate_param it should pass it.

### CODE REVIEW

File: extensions/openai/completions.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def process_parameters(body, is_legacy=False):
         # XXX convert tokens from tiktoken based on requested model
         # Ex.: 'logit_bias': {'1129': 100, '11442': 100, '16243': 100}
         try:
-            encoder = tiktoken.encoding_for_model(generate_params['model'])
+            encoder = tiktoken.encoding_for_model(generate_params['model'] if generate_params['model'] else "llama")
```
[the remark this answers is not in the document]
Comment (Ph0rk0z) [reply extensions/openai/completions.py:107]: Good question. This passes the model name and not the type to tiktoken. So in the case that the "model" param was loaded it would be some string. Let's say it's AuroraNights or Euryale.. that's all tiktoken gets. Have you peeked into the place it fails and what tiktoken matches "startswith" on?

[the remark this answers is not in the document]
Comment (Ph0rk0z) [reply extensions/openai/completions.py:107]: Whoever wrote this must have assumed it was going to be used with openAI and it's pulling double duty. 

so perhaps it's better to just write `if not if generate_params['model'] pass` or something like that.

I will have to test and try it.

**Comment by Ph0rk0z** [comment]
Tiktoken is skipped. Anyone know if that parameter is ever filled? In either case unless it matches the list in tiktoken, the results will be the same and native tokenizer will be used.

**Comment by oobabooga** [comment]
Why not remove tiktoken and use the loaded tokenizer for logit bias? I think that would be a better solution.

**Comment by Ph0rk0z** [comment]
Does anyone use this for OpenAI and will removing the whole block cause issues?

**Comment by oobabooga** [comment]
This PR is called "Fix logits bias in API [completions.py]". The change it makes is to raise an exception if the API call doesn't include a parameter. I am closing it because I don't think it makes sense. Logit bias handling has to be improved in some other way.

**Comment by Ph0rk0z** [comment]
We can just get rid of using tiktoken? Otherwise logit bias doesn't work at all.
